static/src/stylesheets/module/content/_gallery.scss

```diff
@@ -157,8 +157,10 @@
         height: 33px;
         margin: $gs-baseline/3 $gs-gutter/5;
         display: none;
+    }
 
-        .inline-expand-image {
+    &.inline-expand-image {
```

Comment:
just a minor thing, `&.inline-expand-image svg` would probably be preferable to nesting here